code/helpers/overlays.dm

```diff
@@ -0,0 +1,3 @@
+/atom/proc/update_overlays(list/overlay_list)
+	overlays.Cut()
+	overlays += overlay_list
```

Comment:
I would recommend doing this the SS13 way:
```
/atom/proc/update_overlays())
	. = list()
```
then in any object you use update_overlays you do:
```
/obj/whatever/update_overlays())
	. = ..()
	. += "overlay"
```
and in `update_appearance()` you do `overlays.Cut()` and `overlays += update_overlays()`

or the other option,
make the parent `update_overlays` cut the overlays list, and then add to overlays directly in the `update_overlays` proc. This is what 2D Cubemans does.